README.md

```diff
@@ -54,10 +54,10 @@ Other parameters can also be specified as below:
 Donut help
 Usage: Donut reports [options]
 
-  -s <value> | --sourcedirs <value>
-        Use --sourcedirs cucumber:/my/path/cucumber-reports -> Required, or Use --sourcedirs specflow:/my/path/specflow-reports,/my/nunit-reports
-  -o <value> | --outputdir <value>
-        Use --outputdir /my/path/output/donut
+  -s <value> | --sourcePaths <value>
+        Use --sourcePaths gherkin:/my/path/cucumber-reports,gherkin:/my/adapted/nunit-reports
+  -o <value> | --outputPath <value>
+        Use --outputPath /my/path/output/donut

```

Comment:
I suggested updating the readme with the new version after actually doing a release so that users don't get confused as `version 1.1` doesn't follow these formats.
Could you please move any changes in the readme related to the usage of the packaged `donut` to the other PR?
Sorry to be pedantic but I think this is important.